integration/smartcontract/smartcontracts_test.go

```diff
@@ -90,7 +89,7 @@ func TestManagementContract(t *testing.T) {
 		"detectSimpleFork":                   detectSimpleFork,
 	} {
 		t.Run(name, func(t *testing.T) {
-			bytecode, err := managementcontract.Bytecode()
+			bytecode, err := constants.Bytecode()
```

Comment:
I'm not entirely sure where to put this, but it shouldn't be nearby the solidity contracts (albeit closely related)